list.rb

```diff
@@ -0,0 +1,16 @@
+require './enumarable'
+
+class MyList
+  include MyEnumerable
+  def initialize(array)
+    @list = array
+  end
```

Comment:
- [ ] When trying to test your project, I noticed that the `initialize` method hasn't been properly setup to receive multiple arguments and as a result of that, it was causing an error in your project. So kindly fix that :wink: 
- Error Message
![image](https://github.com/mahammad-mostafa/enumerable-methods/assets/102699958/4e2e2905-0011-4ca2-bd7e-8dc224dc46d2)

- [ ] You can rewrite it like so :+1: 
```ruby
    def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```